### DISCUSSION TITLE
posts/my-web-api-spcification/

### DISCUSSION THREAD

**Discussion by giscus** [waynerv/waynerv.github.io#32] [Announcements]
# posts/my-web-api-spcification/

基于 GitHub REST API v3 和 RFC7807，并结合自己在工作中的一些实践，编写了这份 Web API 规范。

https://waynerv.com/posts/my-web-api-spcification/

**Comment by ghost** [comment]
这样的设计非常好：

- 详情不包装
- 列表无论是否分页，为了简洁和一致，均包装
